scripts/check_outputs.py

```diff
@@ -195,11 +203,21 @@ def _check_bold_preprocs(preprocs):
             assert metadata['SkullStripped'] is False
         else:
             img = nb.load(preproc)
-            if '_boldref.' in preproc.name:
-                assert len(img.shape) == 3
-            elif 'desc-preproc_bold' in preproc.name:
-                assert len(img.shape) == 4
-        print(u' \u2713')
+            assert len(img.shape) == 4
+        print(CHECKMARK)
+
+
+def _check_boldrefs(boldrefs):
+    for boldref in boldrefs:
+        print(str(boldref), end='')
+        if '.json' in boldref.name:
+            metadata = json.loads(boldref.read_text())
+            assert 'Sources' in metadata
+        else:
+            bimg = nb.load(boldref)
+            if len(bimg.shape) > 3:
+                assert len(bimg.shape) == 4 and bimg.shape[3] == 1
```

Comment:
This should be standardized - we can either have the BOLD references be 3D or 4D with time dim of 1. Currently, the `desc-hmc` is 3D and `desc-coreg` is 4D. My inclination is just to squeeze into 3D.

cc @effigies @oesteban